binaries/upload.py

```diff
@@ -52,7 +60,9 @@ def upload_conda_packages():
         for name in files:
             file_path = os.path.join(root, name)
             print(file_path)
-            if any(word in file_path for word in PACKAGES) and file_path.endswith("tar.bz2"):
+            if any(word in file_path for word in PACKAGES) and file_path.endswith(
+                "tar.bz2"
+            ):
                 print(f"Uploading to anaconda package: {name}")
                 anaconda_upload_command = f"anaconda -t {anaconda_token} upload --user {CONDA_USER} {file_path} --force"
                 print(f"cmd={anaconda_upload_command}")
```

Comment:
`print(f"cmd={anaconda_upload_command}")` this would leak the tokens to the whole world. It's not as big as a deal in the past because this script was assumed to run on a local machine

Reply:
This function is not being used by the this workflow. But you are right, we shouldn't be printing it. Done